src/index.js

```diff
@@ -92,7 +92,7 @@ function App() {
                   exact
                   path="/ClubDirectorDashboard"
                   component={() =>
-                    Authorization([Roles[0], Roles[1]], ClubDirectorDashboard)
```

Comment:
We don't have ClubDirectorDashboard anymore? Why?

Reply:
because it appears that it was just a stand in page(as it was just a blank page with no content), once we are able to obtain data from all the clubs this wont be hardcoded as Anderson anymore, but instead it can be made more dynamic so that depending on which club is logging in, it displays that club as opposed to just Anderson every time.